prov/util/src/util_buf.c

```diff
@@ -269,8 +270,7 @@ int ofi_bufpool_create_attr(struct ofi_bufpool_attr *attr,
 	pool->region_size = pool->alloc_size - pool->entry_size;
 
 	FI_DBG(&core_prov, FI_LOG_CORE,
-		"%s alloc_size %zu region_size %zu align_entry %zu "
-		"entry_size %zu chuck_cnt %ld pool %p  (%p)\n",
+		"%s alloc_size %zu region_size %zu align_entry %zu entry_size %zu chunk_cnt %ld pool %p  (%p)\n",
```

Comment:
line length is too long, please revert change

Reply:
Yes, I know the kernel guidelines.  The print includes the file and line number anyway, so grep isn't needed.  Besides, it's not like someone can take the print output and toss it into grep anyway.  They have to know how and where to convert portions of the output into a regular expression.

Please keep the typo fix, but not make me need to scroll left/right in my editor.  My eyeballs hate that.